synapse/lib/snap.py

```diff
@@ -684,12 +686,16 @@ async def iterStormPodes(self, text, opts, user=None):
         if opts is not None:
             dorepr = opts.get('repr', False)
             dopath = opts.get('path', False)
+            show_storage = opts.get('show:storage', False)
 
         async for node, path in self.storm(text, opts=opts, user=user):
 
             pode = node.pack(dorepr=dorepr)
             pode[1]['path'] = await path.pack(path=dopath)
 
+            if show_storage:
```

Comment:
Should we make this an argument on `node.pack` since the node will already have all the sodes in 3.0.0?

Reply:
er... `node.getStorNodes()` :)

Reply:
gotcha, yeah that works too